test/forge/helpers/IntegrationTest.sol

```diff
@@ -115,6 +116,18 @@ contract IntegrationTest is BaseTest {
         vault.acceptCap(id);
 
         assertEq(vault.config(id).cap, newCap, "_setCap");
+
+        if (newCap > 0) {
+            if (!wasEnabled) {
+                Id[] memory newSupplyQueue = new Id[](vault.supplyQueueLength() + 1);
+                for (uint256 k; k < vault.supplyQueueLength(); k++) {
+                    newSupplyQueue[k] = vault.supplyQueue(k);
+                }
+                newSupplyQueue[vault.supplyQueueLength()] = id;
+                vm.prank(ALLOCATOR);
+                vault.setSupplyQueue(newSupplyQueue);
+            }
+        }
```

Comment:
Couldn't we have a dedicated function that adds a market to the supply queue ?

Reply:
if you want but it's only used here so Im' not sure that it's useful

Reply:
Indeed overkill for now but perhaps later